docs/proposals/dataset_and_simulation.md

```diff
@@ -0,0 +1,149 @@
+      # the url address of python file
+      url: "./examples/pcb-aoi/singletask_learning_bench/testenv/f1_score.py"
+```
+
+For the convenience of developers, the expected testenv.yaml file is as follows:
+
+- The user fills in the `dataset_url`, here is the HUAWEI CLOUD OBS link.
+- The user fills in `dataset_path`, where the dataset wants to be saved.
+
+```yaml
+testenv:
+  # dataset configuration
+  dataset:
+    # the url address of dataset with zip file; string type;
+    dataset_url: "https://kubeedge.obs.cn-north-1.myhuaweicloud.com:443/ianvs/pcb-aoi/dataset.zip"
+    # the url address of saving dataset
+    dataset_path: "./dataset/"
+
+  # metrics configuration for test case's evaluation; list type;
+  metrics:
+      # metric name; string type;
+    - name: "f1_score"
+      # the url address of python file
+      url: "./examples/pcb-aoi/singletask_learning_bench/testenv/f1_score.py"
+```
+
+The expected implementation method is to modify `dataset.py` in the `testenvmanager` module as follows:
+
+1. Add `self.dataset_url`, `self.dataset_path`
+2. Add `download_save()` function
+
+![](https://github.com/iszhyang/images/blob/main/ianvs/dataset-download.png?raw=true)
+
+### Semantic segmentation dataset and corresponding algorithm
+
+dataset: [cityscapes](https://www.cityscapes-dataset.com)
+Baseline algorithm: [RFNet](https://github.com/AHupuJR/RFNet)
+
+We will provide the cityscapes dataset and corresponding methods for ianvs. The specific operation is as follows:
+
+- dataset processing
+  - generate the index file for train and test
+- dataset download link
+  - provide the download link of cityscapes dataset in HUAWEI Cloud OBS.
+- Baseline algorithm
+  ![](https://github.com/iszhyang/images/blob/main/ianvs/rfnet.png?raw=true)
+
+### Industrial Distributed Collaborative System Simulation
+
+Provide ianvs with the feature of industrial distributed system simulation.
+![img](https://github.com/iszhyang/images/blob/main/ianvs/simulation.jpg?raw=true)
+
+- The `System Config` denotes the system config of the current test case, such as the `number of edge nodes`.
+- The `Simulation Controller` is the core module of system simulation, including `Paradigm Management` and `Enviroment Management`.
+  - `Paradigm Management` is used to manage the paradigm in simulation, including the following
+    - `Incremental Learning Management`
+    - `Lifelong Learning Management`
+    - etc.
+  - `Enviroment Management` is used to build and manage the simulation enviroments.
+    - `Enviroment Builder Controller` is used to parse the system config(simulation), build the simulation enviroment, and finally close the enviroment completely.
+    - `Container Management` manage the containers used to build the simulation enviroment.
+    - `Simulation Job Controller`is responsible for building and managing the job required in the simulation environment.
+
+![img](https://github.com/iszhyang/images/blob/main/ianvs/simulation-dataflow.jpg?raw=true)
```

Comment:
Similarly, the dataflow is overall fine but major revision is needed on the test case and container management of  simulation system.